web-components/src/components/cards/StepCard.tsx

```diff
@@ -221,31 +174,46 @@ function StepCard({
             )}
           </div>
         </div>
```

Comment:
The border got lost somehow:

https://deploy-preview-906--regen-registry.netlify.app/create-methodology

<img width="995" alt="Screen Shot 2022-05-05 at 6 21 52 AM" src="https://user-images.githubusercontent.com/9438198/166922321-45efe376-3ab5-4ffc-9104-3996bc42ac22.png">